ui/src/pages/job/form/components/SecretsForm.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright 2020 The Merlin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Fragment, useCallback, useEffect, useState } from "react";
+
+import {
+  EuiButtonIcon,
+  EuiFieldText,
+  EuiInMemoryTable,
+  EuiTitle,
+  EuiSuperSelect
+} from "@elastic/eui";
+import PropTypes from "prop-types";
+import { useParams  } from "react-router-dom";
+import { useMerlinApi } from "../../../../hooks/useMerlinApi";
+
+require("../../../../assets/scss/Secrets.scss");
+
+export const SecretsForm = ({ variables, onChange }) => {
```

Comment:
This component is more or less a simplified copy of the `EnvironmentVariablesForm` component.